### DISCUSSION TITLE
List of GC products necessary to read for cosmological inference

### DISCUSSION THREAD

**Discussion by gcanasherrera** [euclidlib/euclidlib#37] [General]
List of products/information that we need to retrieve from the Euclid SGS and SWGs to perform the Euclid cosmological inference exercise for Galaxy Clustering Spectroscopic. 

Link to the Euclid DPDD: http://st-dm.pages.euclid-sgs.uk/data-product-doc/dm10/index.html 

- [ ] Pk (must contain fiducial cosmology and effective redshift)
- [ ] nbar (should already be included in the header, we should check)
- [ ] k-binning scheme (for consistency check, we should check)
- [ ] k_effective values (there is a column but values are not stored, request to OU-GC-LE3)
- [ ] CM-Pk
- [ ] 2pcf (must contain fiducial cosmology and effective redshift)
- [ ] prior information for purity parameters

Tagging GC experts @chiaramoretti @alfonso-veropalumbo

**Comment by chiaramoretti** [comment]
Thanks @gcanasherrera . From the Fourier space-side we will also need 

- [ ] nbar (should already be included)
- [ ] info on the k-binning scheme (for consistency check)
- [ ]  k_effective values (there is a column but values are not stored)

To run the analysis we will also need the mixing matrix for window convolution (should have the same structure of the Pk file, see [bottom of this page](http://st-dm.pages.euclid-sgs.uk/data-product-doc/dm10/le3dpd/gc/pk-gc/dpcards/pkgc_OUTPUT_DOA.html#gcpkspefile)

**Reply by gcanasherrera** [reply]
Thanks @chiaramoretti , I have updated the list above

**Comment by gcanasherrera** [comment]
@annmapo about possible missing products interesting for the cosmological inference analysis (thanks!)

**Reply by annmapo** [reply]
Very late reply, sorry!
Some estimate of the galaxy bias values will be important for the covariance matrix (it impacts the error bars). In the case of photometric GC, we will also need the redshift distributions and priors on the shift to the mean and stretch of the width, at least.
